clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-configs
+  namespace: prow-monitoring
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: blackbox
```

Comment:
Why can we not use the prometheus operator custom resources for this?

Reply:
https://github.com/kubernetes/test-infra/pull/15000#issuecomment-547189270

Reply:
Can you add a comment with a link to https://github.com/coreos/prometheus-operator/issues/256 ?

Reply:
Done.